modules/check_in/README.rdoc

```diff
@@ -1,10 +1,15 @@
-= CheckIn
-TODO: Short description and motivation.
+# What is vets-api CheckIn?
```

Comment:
Is this file related to the flipper change?

Reply:
This must have been a merge conflict